Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/colmartech_vendors.yml

```diff
@@ -1933,6 +1933,134 @@
       - id: CMMaskGas
       - id: CMMaskCoif
 
+# MILITARY POLICE
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechMilitaryPoliceEquipment
+  name: ColMarTech Military Police Equipment Rack
+  description: An automated rack hooked up to a colossal storage of Military Police standard-issue equipment.
+  components:
+  - type: AccessReader
+    access:
+    - [ "CMAccessBrig" ]
+  - type: Sprite
+    sprite: _RMC14/Structures/Machines/VendingMachines/ColMarTech/clothing.rsi
+  - type: CMAutomatedVendor
+    job: CMMilitaryPolice
+    sections:
+    - name: Standard Equipment
+      takeAll: CMStandard
+      entries:
+      - id: RMCVendorBundleMilitaryPoliceApparel
+      - id: CMArmorM2MP
+      - id: CMMRE
```

Comment:
Should be split into two sections, some items missing/not correct
![image](https://github.com/RMC-14/RMC-14/assets/121047731/c35cca72-043e-47bb-b641-8ae7980fa192)

Reply:
Also missing below this is the armor and beret options
![image](https://github.com/RMC-14/RMC-14/assets/121047731/b9fd19b6-34f5-46f4-80f6-6666fe0fb939)

Reply:
done.